src/main/java/com/minecolonies/coremod/compatibility/jei/transer/PrivateCraftingTeachingTransferHandler.java

```diff
@@ -0,0 +1,69 @@
+package com.minecolonies.coremod.compatibility.jei.transer;
+
+import com.google.common.collect.ImmutableSet;
+import com.minecolonies.api.util.ItemStackUtils;
+import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.inventory.CraftingGUIBuilding;
+import com.minecolonies.coremod.network.messages.TransferRecipeCrafingTeachingMessage;
+import mezz.jei.api.gui.IGuiIngredient;
+import mezz.jei.api.gui.IGuiItemStackGroup;
+import mezz.jei.api.gui.IRecipeLayout;
+import mezz.jei.api.recipe.transfer.IRecipeTransferError;
+import mezz.jei.api.recipe.transfer.IRecipeTransferHandler;
+import mezz.jei.api.recipe.transfer.IRecipeTransferHandlerHelper;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.inventory.Slot;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.Tuple;
+import net.minecraft.util.text.translation.I18n;
+
+import javax.annotation.Nullable;
+import java.util.*;
+import java.util.stream.Collectors;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.stream.Collectors'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)